lms/djangoapps/teams/static/teams/js/views/team_count_detail.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
CardView is designed to be generic by displaying an array of child views as item details. TopicCardView is a CardView, so it displays the team count through a TeamCountDetailView as a child view.